src/main/java/com/nylas/NylasAccount.java

```diff
@@ -65,10 +65,14 @@ public ContactGroups contactGroups() {
 	public JobStatuses jobStatuses() {
 		return new JobStatuses(client, accessToken);
 	}
-
+  
 	public RoomResources roomResources() {
 		return new RoomResources(client, accessToken);
 	}
+  
+  public Neural neural() {
+		return new Neural(client, accessToken);
+  }
```

Comment:
It is tabbed though? I'm seeing tabs in the snippet above, are you seeing spaces? I'll double check but I thought everything was tabbed because I changed that in my IDE.

Reply:
Yes, looks like this change had spaces instead.